docs/content/doc/features/webhooks.en-us.md

```diff
@@ -188,3 +188,7 @@ if (json_last_error() !== JSON_ERROR_NONE) {
 ```
 
 There is a Test Delivery button in the webhook settings that allows to test the configuration as well as a list of the most Recent Deliveries.
+
+### Authorization header (Gitea hook only)
```

Comment:
Is it still "Gitea hook only"?

Reply:
You are right! (this was the main point of this PR :)

Fixed.